R/drive_cp.R

```diff
@@ -11,12 +11,13 @@
 #' @template name
 #' @templateVar name file
 #' @templateVar default Defaults to "Copy of `FILE-NAME`".
+#' @template dots
```

Comment:
Suspect this should be something more specific, like `dots-metadata`. First, someone might misuse it if they think it's a general dots documentation. Second, we might very well have more than one template re: dots before this all over.